### DISCUSSION TITLE
Document what formats are supported by `pyvista.read()`

### DISCUSSION THREAD

**Discussion by adeak** [pyvista/pyvista#1500] [Ideas]
Reading OpenFOAM data files into pyvista came up in Slack. Ostensibly we support it, since we've got https://github.com/pyvista/pyvista/blob/e9c641c7cbd8047c9adb207146eac1416a3269d0/pyvista/utilities/fileio.py#L56

added in https://github.com/pyvista/pyvista/commit/e7ace4b8f9430073a1373d6f36d7c6462826ae11.

Yet [in a later issue](https://github.com/pyvista/pyvista/issues/924) this seems unresolved, so it's likely that importing doesn't (fully) work properly. Without any experience and hands-on data I can't decide myself.

It would be nice if we could document somewhere (probably in the docs of `pyvista.read()`) what formats are actually supported. I'd prefer a stale hard-coded list to a complete lack of a list. And the nature of input formats is that we'd normally keep supporting a given format once it's in (unless there's a new version of the format with features unaccounted for in PyVista).

**Comment by akaszynski** [comment]
In the docs of `pyvista.read` (with some cool plotting examples) would be best.
